### PR TITLE
DOCSP-30867 Add 1.0 Changelogs

### PR DESCRIPTION
**Pull request by ianf-mongodb**
## Description
Changelogs added to Relational Migrator index page.
Also removed a duplicate ref causing a build error.

## Staging
https://docs-mongodbcom-staging.corp.mongodb.com/docs-relational-migrator/docsworker-xlarge/DOCSP-30867/release-notes/

## Jira
https://jira.mongodb.org/browse/DOCSP-30867

## Build
https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=64925a93259a8493e541bdbe

### CODE REVIEW

**Comment by ianf-mongodb** [comment]
@sarah-olson-mongodb Ready for review.

**Review by sarah-olson-mongodb — APPROVED** [review]
Thanks a ton @ianf-mongodb! Much appreciated. 
LGTM

File: source/projects/configure-settings.txt [hunk 1]
```diff
@@ -73,8 +73,6 @@ inserted into a collection. You can choose one of the following options:
     For example, if your relational table row has a primary key of ``personId = 1``, 
     after migration your MongoDB ``_id`` field is ``_id: ObjectId("62472f3d5849820a91d7b52f")``.
```
Comment (ianf-mongodb) [inline source/projects/configure-settings.txt:75]: **IGNORE**

**Review by tomhollander — CHANGES_REQUESTED** [review]
The first public release will be version 1.1.2, not 1.0.

**Comment by ianf-mongodb** [comment]
Woops, updated @tomhollander

**Review by tomhollander — COMMENTED** [review]
Perfecto

**Pull request merged**